lms/djangoapps/courseware/courses.py

```diff
@@ -593,3 +593,19 @@ def _get_default_child_module(child_modules):
                 child = _get_default_child_module(children)
 
     return child
+
+
+def get_course_chapters(course_key):
```

Comment:
Nit: Can we rename this to `get_course_chapter_ids`, since it doesn't actually return the chapter/SectionDescriptor objects?